### ISSUE TITLE
Life Itself website: get a plan of work Feb 2023

### ISSUE THREAD

**Issue by rufuspollock** [life-itself/community#334]
[Created: Feb 2] We have successfully launched the new site. We need to clarify top level plan of work going forward plus specific next steps. Top level should be in #170 but that needs some tidying up.

## Acceptance

- [x] Have up to date plan of work in one place **🚧2023-02-16 #170 is one place. Updating plan is in progress and consolidating material there. 60% done** **✅2023-03-10 as of now**
- [x] Have project summary in one place with consolidated material there **🚧2023-02-16 A10 is it** **✅2023-03-10 for now**

### Meta

- [x] Identify where is the top level plan of work for life itself website? **✅2023-02-02 this should be in #170**
- [x] Is this plan of work up to date? **✅2023-02-02 not completely. Added action below**
- [x] Is there stuff in other issues that should be "hoisted" into epic e.g. stuff now in https://github.com/life-itself/community/issues/260? **✅ 2023-02-06 have hoisted into epic and left #260 with immediate actions only // 2023-02-02 yes probably and we can work on this**
- [x] What's the duplication between A10 and epic issue? **✅2023-02-02 main duplication was job stories and some motivation etc at the top. We've remove this for now. Other duplication was goals in purpose/outcome visioning in A10. We can keep these there and copy some to acceptance in epic**
- [x] Where do we keep the overall project plan? Is it the A10, is it a "project file" in `/notes/projects/xxx`? is it the epic issue? **🚧2023-02-02 we won't create projects/xxx for now. A10 has overview. Epic has tasks and acceptance**
  - [x] Where do key goals / acceptance live? in A10 or in epic or ... **✅2023-02-02 in epic**
  - [x] where does summary / context for the project live? A10 or in epic or project plan in obsidian?  **✅2023-02-02 in A10**
  - [x] where do key steps / tasks live? **✅2023-02-02 in the epic (unless we get radical and move this stuff into obsidian)**
  - [x] Should we partition technical "upgrade/migration" to Flowershow from "updating content" **✅2023-02-02 ❌ no**
- [x] What can we consolidate of Website Restructure 2022 Planning (and if processed can we delete or mark as deprecate this old doc) @nathen **✅ 2023-02-06 has been consolidated into issue #170 and #223** https://docs.google.com/document/d/1y_CG4rdWO7IvCEsMux9lVRO_ko_BTHWAZetTe9-CD9I/edit 

### Get plan of work

- [x] Consolidate into #170
  - [x] Consolidate initiative planning work (in gdoc) https://docs.google.com/document/d/17yPmLPOBzT1mVhpqeirEekNGDEzQ2hi_uyeTKliPvKA/edit#heading=h.ig183qro1hv1 into the issue https://github.com/life-itself/community/issues/97 @nathen **✅ all moved into the issue and document ~~deprecated~~ deleted**
  - [x] Reduce next steps issue (#260) to essentials and hoist material to #170 **✅2023-02-16 done this ~rufus** 
  - [x] Clean up tasks list to be just inbox etc **✅2023-02-16**
  - [x] Move ecosystem II stuff #183 into #170 task list and delete issue (given that it is a small set of misc tasks that are mainly questions) **✅2023-02-16 by rufus**
  - [x] Review #223 to see if this can be consolidated etc (how does it relate to job stories etc) **✅2023-03-10 nothing really to refactor up and worth keeping as an issue as quite a bit there. Probably sequence after higher priority stuff like projects and front page**
- [x] Refactor acceptance list to reflect current high level plan of work **🚧2023-02-16 getting close** **✅2023-03-10 all done**
- [x] Review and analyze already identified high priority issues
  - [x] Clarify #97 **🚧2023-02-16 did enough to unblock @nathenf to make a start i think**
  - [x] Clarify #335  - have outline of front page
- [x] Review inbox and select any which are to be worked on next
- [x] Analyze those and create issues
- [x] Estimate
- [x] Schedule

### Misc (nice to have)

- [x] Move life itself wireframing to https://excalidraw.com/#room=cbed68bdcd0ef72c78b5,BHedlhnS9SpLQ_2pb_Mv7Q to app.excalidraw and give access @rufus **✅2023-03-10 https://link.excalidraw.com/l/9u8crB2ZmUo/3dT2At78h5O**

### Meeting 9 Feb

Present: Nathen, Rufus

Agenda

- [x] Ready to create initiatives page how do I do this? **✅2023-02-16 ❌ this is deprioritized pending other items e.g. job stories**
- [x] For Tao logos how do I force image to save as when clicking on it. **Rufus: what is the "so that". I don't think we need to force save images.**
- [x] double check website structure planning material was migrated and where
https://docs.google.com/document/d/1y_CG4rdWO7IvCEsMux9lVRO_ko_BTHWAZetTe9-CD9I/edit#heading=h.b7wek4qtzbbb **✅2023-02-09**
- [x] work on job stories **🚧2023-02-09 did a first pass**
- [x] work on plan for front page **🚧2023-02-09 started on this. created issue #XXX**

**Comment by rufuspollock** [comment]
@nathenf this is something we could focus on in our next call as its the prep work for website plan.

**Comment by nathenf** [comment]
@rufuspollock  I think we can close this issue. Do you agree?

**Comment by rufuspollock** [comment]
@nathenf i wasn't sure we have fully analysed the main next issues and got estimates. once we have done that we can close.

**Comment by rufuspollock** [comment]
FIXED. We have project thoroughly analysed and quite a bit of the new site as well.
